docs/release-notes/1.4.0.asciidoc

```diff
@@ -0,0 +1,42 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-1.4.0]]
+== {n} version 1.4.0
+
+
+
+[[feature-1.4.0]]
+[float]
+=== New features
+
+* Add Elastic Agent CRD (standalone mode) {pull}4010[#4010]
+* Add support for multi-arch container images  {pull}3897[#3897] (issue: {issue}3504[#3504])
+
+[[enhancement-1.4.0]]
+[float]
+=== Enhancements
+
+* Add support for a user-defined transport CA certificate and key {pull}4053[#4053] (issue: {issue}2812[#2812])
+* Add support for DaemonSetUpdateStrategy {pull}4049[#4049] (issue: {issue}3839[#3839])
+* [Helm] Use ValidatingWebhookConfiguration v1 on supported Kubernetes versions {pull}3963[#3963] (issues: {issue}3886[#3886], {issue}3958[#3958])
+* Include Beats daemonsets in the diagnostic bundle {pull}3908[#3908]
+* Add support for webhook constraints {pull}3876[#3876] (issue: {issue}3431[#3431])
+* Remove unnecessary metrics from _nodes ES API request {pull}3860[#3860] (issue: {issue}3249[#3249])
+* Allow automatic Elasticsearch nodes discovery {pull}3837[#3837] (issues: {issue}3182[#3182], {issue}3723[#3723])
+* Enable webhook dry run on supported k8s versions {pull}3834[#3834] (issue: {issue}1233[#1233])
```

Comment:
```suggestion
* Enable webhook dry run on supported Kubernetes versions {pull}3834[#3834] (issue: {issue}1233[#1233])
```

Reply:
I will address this in #4116 (I should have mentioned that this PR is based off of #4116, sorry)